src/otx/algo/classification/vit.py

```diff
@@ -573,22 +577,6 @@ def _customize_outputs(
             labels=preds,
         )
 
-    @property
```

Comment:
Could you explain why it's removed?

Reply:
fixed at cad98616c8490ef00997c98da5224da1c4b2da50